docs/installation.md

```diff
@@ -16,10 +16,10 @@ The following instructions are for setting up a version of Amundsen using Docker
     # For Atlas
     $ docker-compose -f docker-amundsen-atlas.yml up
     ```
-4. Ingest dummy data into Neo4j by doing the following: _(Please skip if you are using Atlas backend)_
+4. Ingest provided sample data into Neo4j by doing the following: _(Please skip if you are using Atlas backend)_
 
-   * Change directory to the [amundsendatabuilder](https://github.com/lyft/amundsendatabuilder) submodule.
-   * Run the following commands in the `amundsendatabuilder` upstream directory:
+   * In a separate terminal window, change directory to the [amundsendatabuilder](https://github.com/lyft/amundsendatabuilder) submodule.
+   * `sample_data_loader` python script included in `examples/` directory uses _elasticsearch client_, _pyhocon_ and other libraries. Install the dependencies in a virtual env and run the script by following the commands below:
    ```bash
     $ python3 -m venv venv
     $ source venv/bin/activate  
```

Comment:
thanks @Alagappan , but wouldn't L24-L25 suggested creating virtual env?

Reply:
yes, the command work as expected. It wasn't clear why we need to install the dependencies again separately (and requirements.txt)  when we have docker container running already from step 3. Just updated the instructions to clarify we only need this step for sample script.

Reply:
thanks